frontend/src/components/Plan/components/StatementSection/ReleaseView/ReleaseView.vue

```diff
@@ -203,16 +201,24 @@ const getChangeTypeText = (changeType: Release_File_ChangeType) => {
       return t("release.change-type.ddl-ghost");
     case Release_File_ChangeType.DML:
       return t("release.change-type.dml");
+    case Release_File_ChangeType.CHANGE_TYPE_UNSPECIFIED:
+      return t("release.change-type.unspecified");
     default:
+      changeType satisfies never;
```

Comment:
Do we need this line? (same for line 221

Reply:
for exhaustive check